packages/docusaurus-theme-classic/src/theme/BlogPostPage/index.tsx

```diff
@@ -64,7 +75,9 @@ export default function BlogPostPage(props: Props): JSX.Element {
           ThemeClassNames.page.blogPostPage,
         )}>
         <BlogPostPageMetadata />
-        <BlogPostPageContent sidebar={props.sidebar}>
+        <BlogPostPageContent
+          sidebar={props.sidebar}
+          blogMetadata={props.blogMetadata}>
```

Comment:
if the place we render structured data doesn't matter, why not doing it here instead of props drilling the blog metadata?

Reply:
Because moving it further up results in this error:

`Hook useBlogPost is called outside the <BlogPostProvider>. `

![image](https://github.com/facebook/docusaurus/assets/1010525/376de48f-ef95-4209-96d4-6cca1d3563c6)

Reply:
Ignore - now sorted